lib/pf9-builder/testcase-01-ubuntu.csv

```diff
@@ -0,0 +1,8 @@
+## openstack
```

Comment:
Please remove these files from the pull request, we should ultimately distribute internally and add to the gitignore